packages/synapse-react-client/src/components/CertificationQuiz/CertificationQuiz.tsx

```diff
@@ -94,135 +126,189 @@ const CertificationQuiz: React.FunctionComponent = () => {
+                          sx={{
+                            table: {
+                              textAlign: 'center',
+                              th: {
+                                padding: '5px',
+                              },
+                            },
+                          }}
+                          actionButton={actionButtonConfig(question.docLink)}
+                          showCloseButton={true}
+                        >
+                          <Typography variant="hintText" color="primary">
+                            <HelpOutlineTwoTone
+                              className="HelpButton"
+                              style={{ marginRight: '4px' }}
+                            />
+                            Need help answering this question?
+                          </Typography>
+                        </MarkdownPopover>
+                      )}
+                    </li>
+                  )
+                })}
+              </ol>
+            </form>
+            <Button
+              className="help-button"
+              color="primary"
+              variant="contained"
+              size="large"
+              disabled={isSubmitting}
+              onClick={() => {
+                handleSubmit()
+              }}
+            >
+              Submit
+            </Button>
+          </>
         )}
-        <form ref={formRef} onSubmit={e => e.preventDefault()}>
+
+        {!isTakingQuiz && passingRecord?.corrections && (
           <ol>
-            {quiz?.questions.map(question => (
-              <li
-                key={question.questionIndex}
-                role={question.exclusive ? 'radiogroup' : undefined}
-              >
-                <div
-                  dangerouslySetInnerHTML={{ __html: question.prompt }}
-                  className={
-                    passingRecord &&
-                    (passingRecord.corrections?.find(
-                      quest =>
-                        quest.question.questionIndex === question.questionIndex,
-                    )?.isCorrect
-                      ? ''
-                      : 'incorrectQuestion')
-                  }
-                ></div>
-                {question.answers.map(choice => (
-                  <div key={`${question.questionIndex}-${choice.answerIndex}`}>
-                    <input
-                      id={`${question.questionIndex}-${choice.answerIndex}`}
-                      name={`${question.questionIndex}`}
-                      type={question.exclusive ? 'radio' : 'checkbox'}
-                      value={choice.answerIndex}
-                      onClick={e =>
-                        onUpdateAnswer(
-                          question.questionIndex,
-                          Number(e.currentTarget.value),
-                        )
-                      }
-                      disabled={!!passingRecord}
+            {passingRecord.corrections.map(responseCorrectness => {
```

Comment:
Huge change.  Use the questions stored in the passingRecord, not the fresh Quiz that we received!

Reply:
Before this change, some questions in the fresh quiz appeared to be answered, and some didn't (because the response answer was outside the new question answer set!)